package.json

```diff
@@ -7,8 +7,9 @@
   "license": "CC-BY-SA-3.0",
   "keywords": [],
   "dependencies": {
-    "@material-ui/core": "^3.5.1",
+    "@material-ui/core": "^3.9.4",
     "@material-ui/icons": "^3.0.1",
+    "@material-ui/styles": "^4.7.1",
```

Comment:
Tu utilise cette dépendence?
Je ne pense pas que ce soit utilse sachant qu'on utilise déjà `styled-component` pour le CSS in JS

Reply:
J'ai du l'ajouter lors d'un test. Je nettoie le package.json en supprimant les dépendances inutiles

Reply:
Je nettoie le package.json pour supprimer les dépendances inutiles avant de rebase